addons/hr_timesheet/__manifest__.py

```diff
@@ -50,6 +50,9 @@
         'web.assets_backend': [
             'hr_timesheet/static/src/**/*',
         ],
+        'web.assets_frontend': [
```

Comment:
Shouldn't it move into `project.webclient` ?

Reply:
Done! Thanks :)